components/cli/pkg/runtime/runtime.go

```diff
@@ -338,3 +355,56 @@ func (runtime *CelleryRuntime) Validate() error {
 	}
 	return nil
 }
+
+func (runtime *CelleryRuntime) UnmarshalHelmValues(chartName string) error {
+	defaultValues, err := util.GetHelmChartDefaultValues(chartName)
+	if err != nil {
+		log.Printf("Unmarshal error: %v", err)
+		return err
+	}
+	if chartName == "cellery-runtime" {
+		err = yaml.Unmarshal([]byte(defaultValues), &runtime.celleryRuntimeVals)
+		if err != nil {
+			log.Printf("Unmarshal error: %v", err)
+			return err
+		}
+	}else if chartName == "ingress-controller" {
+		err = yaml.Unmarshal([]byte(defaultValues), &runtime.ingressControllerVals)
+		if err != nil {
+			log.Printf("Unmarshal error: %v", err)
+			return err
+		}
+	}
+	return nil
+}
+
+func (runtime *CelleryRuntime) MarshalHelmValues(chartName string) error {
+	if chartName == "cellery-runtime" {
+		templateYamls, err := yaml.Marshal(&runtime.celleryRuntimeVals)
+		if err != nil {
+			log.Printf("Marshal error: %v", err)
+			return err
+		}
+		runtime.celleryRuntimeYaml = string(templateYamls)
+	} else if chartName == "ingress-controller" {
+		templateYamls, err := yaml.Marshal(&runtime.ingressControllerVals)
+		if err != nil {
+			log.Printf("error: %v", err)
+			return err
+		}
+		runtime.ingressControllerYamls =  string(templateYamls)
+	}
+	return nil
+}
+
+func (runtime *CelleryRuntime) CreateNameSpace(namespace string) error {
```

Comment:
There is already an interface function called "CreateNameSpace" in KubeCli interface to create namespace. Shall we use that?